### PR TITLE
Add `Playtesters` page

### PR DESCRIPTION
**Pull request by BlueChinchompa**
I don't have the dates when people became testers so @Lekuruu is probably going to have to help with that.

Also @Meruru-chan can make a commit and add when they became a tester.

### CODE REVIEW

File: wiki/People/Testers/en.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Testers
+
+Testers are people who have helped test out clients and features on Titanic!
```
Comment (iSlodinxOsu) [inline wiki/People/Testers/en.md:3]:
```suggestion
Testers are users who have helped test out clients and features on Titanic!
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]
Does this mean that they also find bugs on clients aswell?

**Comment by Lekuruu** [comment]
They technically are "playtesters" as soon as they registered, so I'd use that

**Comment by iSlodinxOsu** [comment]
> They technically are "playtesters" as soon as they registered, so I'd use that

Do you agree by replacing N/A to "Since the Beginning"?

**Comment by Lekuruu** [comment]
Uhhh, I don't see a reason why using the registration date would be harder

**Comment by BlueChinchompa** [comment]
> They technically are "playtesters" as soon as they registered, so I'd use that

Im assuming people like rory or crazy were testers since they joined then right?

**Comment by iSlodinxOsu** [comment]
any progress?

**Comment by BlueChinchompa** [comment]
> any progress?

I just added the dates for Natasha and CrazyCSIW6

**Comment by BlueChinchompa** [comment]
Okay this should be ready to be merged now

**Review by iSlodinxOsu — APPROVED** [review]

File: wiki/People/Testers/en.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+[ais](https://osu.titanic.sh/u/3) | July 1st, 2023
+[Natasha](https://osu.titanic.sh/u/5) | October 7th, 2023
+[CrazyCSIW6](https://osu.titanic.sh/u/7)| July 25th, 2023
+[bazded](https://osu.titanic.sh/u/8) | July 28th, 2023
+[Rory](https://osu.titanic.sh/u/9)| July 28th, 2023
+[db2k](https://osu.titanic.sh/u/10) | July 28th, 2023
+[daniel](https://osu.titanic.sh/u/11) | July 29th, 2023
+[Nevy](https://osu.titanic.sh/u/32) | August 11th, 2023
+[Danie](https://osu.titanic.sh/u/37) | August 21st, 2023
+[Meru](https://osu.titanic.sh/u/41) | September 17th 2023
```
Comment (iSlodinxOsu) [inline wiki/People/Testers/en.md:19]:
```suggestion
[Meru](https://osu.titanic.sh/u/41) | September 17th, 2023
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

**Review by iSlodinxOsu — APPROVED** [review]

**Review by Meruru-chan — APPROVED** [review]
w

**Pull request merged**